wordSearch.c

```diff
@@ -11,15 +11,17 @@
 
 #define NUM_WORDS 14
 #define FILE_WORD_COUNT 3000
+#define MAX_WORD_LENGTH 16 // Longest word in file is AFRICANAMERICAN
 
 //#define DEBUG
 
-// TODO: line 86, optimize program so it doesn't read in the file every time
 // TODO: Your mom
 
-void wordPick(FILE *fp, char board[DIM_X][DIM_Y]);
+void wordPick(FILE *fp, char board[DIM_X][DIM_Y],
+              char words[FILE_WORD_COUNT][MAX_WORD_LENGTH]);
 int  validPick(char board[DIM_X][DIM_Y], char wordBuff[BUFSIZ], int x, int y,
                int length, int type, int direction);
+void loadWords(FILE *wordFile, char words[FILE_WORD_COUNT][MAX_WORD_LENGTH]);
```

Comment:
created a new function that specifically loads words from the file into `words[][]`